apps/challenges/views.py

```diff
@@ -106,6 +106,33 @@ def challenge_list(request, challenge_host_team_pk):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def challenge_list_participant(request, participant_team_pk):
+    """
+    Returns lists of callenges that the participant team has participated in.
```

Comment:
Please change `Returns lists of callenges that the participant team has participated in.` ->
`Returns a challenge list in which a participant team has participated`